packages/driver/src/systemUtils.ts

```diff
@@ -1,35 +1,23 @@
 import * as crypto from "node:crypto";
 import { promises as fs } from "node:fs";
-import net from "node:net";
-import os from "node:os";
 import path from "node:path";
-import * as tls from "node:tls";
-
 import process from "node:process";
 import * as readline from "node:readline";
 import { Writable } from "node:stream";
 
-export { path, net, fs, tls, process };
-
 export async function readFileUtf8(...pathParts: string[]): Promise<string> {
   return await fs.readFile(path.join(...pathParts), { encoding: "utf8" });
 }
```

Comment:
`readFileUtf8` is so trivial -- why do we need it? Why not just inline the actual call into whatever uses `readFileUtf8`?

Reply:
we can, I mostly tried to remove deno stuff and not change too much the organisation of code that is not related to that... we use `readFileUtf8` in multiple files and in some files multiple times so I guess it was/is easier to not repeat the `{ encoding: "utf8" } ` part every time

Reply:
I guess we don't strictly need it, but most places where we read files we're joining some path onto a base path, then reading as utf8, so this util makes it look a bit cleaner.

Reply:
> mostly tried to remove deno stuff and not change too much the organisation of code that is not related to that

Yeah, this PR is probably sprawing enough as it is 😄.